### PR TITLE
Overhead/sentry and newrelic

### PR DESCRIPTION
**Pull request by cmac1000**

### CODE REVIEW

**Comment by cmac1000** [comment]
ah, this isn't quite ready, now that I think about it, it needs confd...

File: etc/confd/templates/newrelic.ini.tmpl [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+#    http://docs.python.org/library/configparser.html
+#
+# For further discussion on the behaviour of the Python agent that can
+# be configured via this configuration file see:
+#
+#    http://newrelic.com/docs/python/python-agent-configuration
+#
+
+# ---------------------------------------------------------------------------
+
```
Comment (cmac1000) [inline etc/confd/templates/newrelic.ini.tmpl]: it's a bit weird to put a newrelic license file into source control like this, especially for future users who won't need it (for that matter it's a bit weird to have newrelic and sentry as python dependencies). I'm not sure what a better alternative is, though, and other newrelic users *could* safely use this...

**Comment by cmac1000** [comment]
there are a few WIP commits in this one if it looks good let me know, I'll squash before merging.

File: etc/confd/templates/newrelic.ini.tmpl (hunk 1 above)
Comment (wamberg) [reply etc/confd/templates/newrelic.ini.tmpl]: I agree it's strange. Maybe we'll find a better way to open source applications, and hide deployment details, as we spend more time writing OSS. Maybe we'll find that showing deployment details isn't all that bad.

**Comment by wamberg** [comment]
Squash and self-merge, please.

**Comment by cmac1000** [comment]
closing in favor of squashed version #90

**Pull request closed without merging**